docs/raw/serving-community-history.md

```diff
@@ -0,0 +1,406 @@
+
+The community owner node MUST seed the [generated torrent](#creating-message-archive-torrents) until a new message history archive is created.
+
+The community owner node SHOULD NOT seed torrents for older message history archives. Only one torrent at a time should be seeded.
+
+## Creating magnet links
+
+Once a torrent file for a message archive is created, the community owner node MUST derive a magnet link following the [Magnet URI scheme](https://en.wikipedia.org/wiki/Magnet_URI_scheme) using the underlying BitTorrent protocol client.
+
+## Message archive distribution
+
+Message archives are available via the BitTorrent network as they are being [seeded by the community owner node](#seeding-message-history-archives).
+Other community member nodes will download the message archives from the BitTorrent network once they receive a magnet link that contains a message archive index.
+
+The community owner node MUST send magnet links containing message archives and the message archive index to a special community channel. The topic of that special channel follows the following format:
+
+```
+/{application-name}/{version-of-the-application}/{content-topic-name}/{encoding}
+```
+
+All messages sent with this topic MUST be instances of `ApplicationMetadataMessage` ([6/PAYLOADS](/specs/6-payloads)) with a `payload` of `CommunityMessageArchiveIndex`.
+
+Only the community owner MAY post to the special channel. Other messages on this specified channel MUST be ignored by clients.
+Community members MUST NOT have permission to send messages to the special channel.
+However, community member nodes MUST subscribe to special channel to receive waku messages containing magnet links for message archives.
+
+## Canonical message histories
+
+Only community owners are allowed to distribute messages with magnet links via the special channel for magnet link exchange. Community members MUST NOT be allowed to post any messages to the special channel.
+
+Status nodes MUST ensure that any message that isn't signed by the community owner in the special channel is ignored.
+
+Since the magnet links are created from the community owner node's database (and previously distributed archives), the message history provided by the community owner becomes the canonical message history and single source of truth for the community.
+
+Community member nodes MUST replace messages in their local databases with the messages extracted from archives within the same time range. Messages that didn't receive the community owner node MUST be removed and are no longer part of the message history of interest, even if it already existed in a community member node's database.
+
+
+## Fetching message history archives
+
+Generally, fetching message history archives is a tree step process:
+
+1. Receive message archive index magnet link as described in [Message archive distribution], download `index` file from torrent, then determine which message archives to download
```

Comment:
I'd suggest linking to the "Creating message archive torrents" section for this part:
> download `index` file from torrent